security/org-account-context/main.tf

```diff
@@ -64,6 +64,223 @@ provider "aws" {
   }
 }
 
+####################################################################################################################
+# Following providers are needed to deploy Resource Explorer in all available regions
```

Comment:
because we actually have resources deployed before we managed to introduce SCP and we want to have an overview of what have been deployed so we can do something about it.